packages/sdks/snippets/angular-ssr/src/app/advanced-child/custom-tabs/custom-tabs.component.ts

```diff
@@ -15,46 +15,32 @@ import type {
   standalone: true,
   imports: [CommonModule, Blocks],
   template: `
-    <div>
-      <h2>Custom Component with editable regions</h2>
+    <ng-container *ngIf="tabList?.length">
+      <button
+        *ngFor="let tab of tabList; let i = index"
+        [class.active]="activeTab === i"
+        (click)="activeTab = i"
+      >
+        {{ tab.tabName }}
+      </button>
 
-      <div>
-        <button
-          *ngFor="let tab of tabList; let i = index"
-          [class.active]="activeTab === i"
-          (click)="activeTab = i"
-        >
-          {{ tab.tabName }}
-        </button>
-      </div>
-
-      <div *ngIf="tabList?.length">
-        <div *ngFor="let tab of tabList; let i = index">
-          <div [style.display]="activeTab === i ? 'block' : 'none'">
-            <blocks
-              [blocks]="tabList[i].children"
-              [path]="'component.options.tabList.' + i + '.children'"
-              [parent]="builderBlock.id"
-              [context]="builderContext"
-              [registeredComponents]="builderComponents"
-            ></blocks>
-          </div>
-        </div>
-      </div>
-    </div>
+      <ng-container *ngFor="let tab of tabList; let i = index">
+        <ng-container *ngIf="activeTab === i">
+          <blocks
+            [blocks]="tabList[i].blocks"
+            [path]="'tabList.' + i + '.blocks'"
+            [parent]="builderBlock.id"
+            [context]="builderContext"
+            [registeredComponents]="builderComponents"
+          />
+        </ng-container>
+      </ng-container>
+    </ng-container>
```

Comment:
this can simplify down to:

```suggestion
          <blocks
            [blocks]="tabList[activeTab].blocks"
            [path]="'tabList.' + activeTab + '.blocks'"
            [parent]="builderBlock.id"
            [context]="builderContext"
            [registeredComponents]="builderComponents"
          />
```